docs/local-testing.md

```diff
@@ -47,10 +47,19 @@ This will take a bit longer the first time it is run, as it has to build a chroo
 cd $CHALDIR/test-1
 make test-docker
 sudo apt-get install -y netcat
-nc 127.0.0.1 1337
+nc 127.0.0.1 external_port
```

Comment:
What do you think about changing external_port to [external_port], the same way as [name_of_the_challenge]?